directory.go

```diff
@@ -132,6 +132,20 @@ func (dir *Directory) GetGroups(criteria Criteria) (*Groups, error) {
 	return dir.Groups, nil
 }
 
+//GetAccounts returns all the accounts from a directory
+func (dir *Directory) GetAccounts(criteria Criteria) (*Accounts, error) {
```

Comment:
This is already part of the directory and other resources that hold Accounts, look at accountStoreResource
in resources.go https://github.com/jarias/stormpath-sdk-go (by the way the documentation on that function is incorrect)